src/trunk/libs/seiscomp3/io/recordstream/descriptions/global_recordstream.rst

```diff
@@ -105,14 +105,27 @@ Examples
 SDSArchive
 ----------
 
-This RecordStream reads data from an SeisComP (SDS) archive using the
-:ref:`rs-file` RecordStream. The source is interpreted as a directory path.
+This RecordStream reads data from one or more SeisComP (SDS) archive using the
+:ref:`rs-file` RecordStream. The source is interpreted as a directory path list
+separated using comas.
 
 Example
 ^^^^^^^
 
 - ``sdsarchive:///home/sysop/seiscomp3/var/lib/archive``
 
+- ``sdsarchive:///SDSA,/SDSB,/SDSC``
+
+different SDS are not merged, but are read sequentially depending on data 
+existance. When a missing file in current SDS is requested it tries to read from
```

Comment:
exist**e**nce. If a requested file is missing in the current SDS it is searched in the next archive in the list. On success .. , on failure ...